.github/workflows/plugins_react_tests.yml

```diff
@@ -45,11 +48,11 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node }}
-      # We could update the postinstall action for foreman to look for an environment variable for plugin webpack dirs
-      # before kicking off the ruby script to find them, this would eliminate the ruby dep and running `npm install` in plugins.
-      - uses: ruby/setup-ruby@v1
+      - name: "Set up Ruby ${{ matrix.ruby }}"
+        uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
```

Comment:
Curious, why did you add this? At this point, neither Foreman nor a Plugin is checked out yet, so bundler won't do anything, as there is no Gemfile.

Reply:
Ah yes, but there it happens after Foreman and was checked out, so there is a Gemfile
You can try doing
```yaml
      - name: "Set up Ruby ${{ matrix.ruby }}"
        uses: ruby/setup-ruby@v1
        with:
          ruby-version: ${{ matrix.ruby }}
          bundler-cache: true
          working-directory: ${{ github.workspace }}/projects/foreman
```
*after* the "Set up plugin in Foreman" step.

Reply:
do the JS tests even boot rails? (if they don't, there is no need for the libvirt gems etc to be present)